tests/event_store/test_snapshots.py

```diff
@@ -55,11 +55,12 @@ def test_rejects_snapshot_with_incorrect_version(
 ) -> None:
     given.stream(stream_id := StreamId())
     given.event(AnEvent(), on=stream_id)
+    current_version = given.stream(stream_id).current_version or 0
```

Comment:
it's just for type annotations. The current version can be None when the stream has no events. To avoid mypy issues I've added `or 0`. But it should not be here. Maybe it's better to manually write the version...